packages/react-components/src/components/MessageStatusIndicator.tsx

```diff
@@ -4,6 +4,7 @@
 import { ICalloutContentStyles, Icon, mergeStyles, TooltipHost } from '@fluentui/react';
 import { MessageStatus, _formatString } from '@internal/acs-ui-common';
 import React, { useState } from 'react';
+import { LiveMessage } from 'react-aria-live';
```

Comment:
Just a note for future reference:
Noting that LiveMessage needs to be wrapped in a LiveAnnouncer for it to work, according to the docs: 
https://www.npmjs.com/package/react-aria-live

Followed the trail and found that MessageThread.tsx has LiveAnnouncer which wraps MessageStatusIndicator like so:
\<LiveAnnouncer\> 
**->** \<Chat\> 
**->** item=MessageToDisplay 
**->** itemOnRenderMessageStatus 
**->** memoizeAllMessages 
**->** {message: content: ... defaultStatusRenderer} 
**->** \<MessageStatusIndicator\> 
**->** \<LiveMessage\>

(As of June 2, 2022, things may have changed in between the time you read this comment)